services/net/syndication/SyndicationAction.cs

```diff
@@ -318,27 +326,18 @@ private SourceContent CreateSourceContent(IngestModel ingest, SyndicationItem it
     private async Task<SyndicationFeed> GetFeedAsync(Uri url, IIngestServiceActionManager manager)
     {
         var response = await _httpClient.GetAsync(url);
-        var data = await response.Content.ReadAsStringAsync();
-
         if (!response.IsSuccessStatusCode)
         {
             var ex = new HttpRequestException(response.ReasonPhrase, null, response.StatusCode);
             this.Logger.LogError(ex, "Failed to fetch syndication feed for ingest '{name}'", manager.Ingest.Name);
             throw ex;
         }
 
+        var data = await response.Content.ReadAsStringAsync();
+        var xmlr = XmlReader.Create(new StringReader(data));
```

Comment:
Why did you move this here?  If there is a failure it won't fall through into the catch.

Reply:
Updated, thanks